src/Imath/ImathEuler.h

```diff
@@ -166,163 +175,205 @@ template <class T> class Euler : public Vec3<T>
 
-    //------------------------------------------------------------
-    //	Conversions, toXYZVector() reorders the angles so that
-    //  the X rotation comes first, followed by the Y and Z
-    //  in cases like XYX ordering, the repeated angle will be
-    //	in the "z" component
-    //
-    // The Euler-from-matrix extract() functions assume that the
-    // matrix does not include shear or non-uniform scaling, but
-    // the extract() functions do not examine the matrix to verify
-    // this assumption.  If necessary, you can adjust the matrix
-    // by calling the removeScalingAndShear() function, defined
-    // in ImathMatrixAlgo.h.
-    //------------------------------------------------------------
+    /// @}
+    
+    /// @{
+    /// @name Assignments and Conversions
+    ///
+
+    /// Assignment
+    IMATH_HOSTDEVICE IMATH_CONSTEXPR14 const Euler<T>& operator= (const Euler<T>&) noexcept;
+
+    /// Assignment
+    IMATH_HOSTDEVICE IMATH_CONSTEXPR14 const Euler<T>& operator= (const Vec3<T>&) noexcept;
 
+    /// Assign from Matrix33, assumed to be affine
     IMATH_HOSTDEVICE void extract (const Matrix33<T>&) noexcept;
+
+    /// Assign from Matrix44, assumed to be affine
     IMATH_HOSTDEVICE void extract (const Matrix44<T>&) noexcept;
+
+    /// Assign from Quaternion
     IMATH_HOSTDEVICE void extract (const Quat<T>&) noexcept;
+
+    /// Convert to Matrix33
     IMATH_HOSTDEVICE Matrix33<T> toMatrix33() const noexcept;
+
+    /// Convert to Matrix44
     IMATH_HOSTDEVICE Matrix44<T> toMatrix44() const noexcept;
+
+    /// Convert to Quat
     IMATH_HOSTDEVICE Quat<T> toQuat() const noexcept;
-    IMATH_HOSTDEVICE Vec3<T> toXYZVector() const noexcept;
 
-    //---------------------------------------------------
-    //	Use this function to unpack angles from ijk form
-    //---------------------------------------------------
+    /// Reorder the angles so that the X rotation comes first,
+    ///  followed by the Y and Z in cases like XYX ordering, the
+    ///  repeated angle will be in the "z" component
+    IMATH_HOSTDEVICE Vec3<T> toXYZVector() const noexcept;
 
+    /// @}
+    
+    /// @{
+    /// @name Utility Methods
+    
+    /// Unpack angles from ijk form
     IMATH_HOSTDEVICE void angleOrder (int& i, int& j, int& k) const noexcept;
 
-    //---------------------------------------------------
-    //	Use this function to determine mapping from xyz to ijk
-    // - reshuffles the xyz to match the order
-    //---------------------------------------------------
-
+    /// Determine mapping from xyz to ijk (reshuffle the xyz to match the order)
     IMATH_HOSTDEVICE void angleMapping (int& i, int& j, int& k) const noexcept;
 
-    //----------------------------------------------------------------------
-    //
-    //  Utility methods for getting continuous rotations. None of these
-    //  methods change the orientation given by its inputs (or at least
-    //  that is the intent).
-    //
```

Comment:
Did these lines disappear?

You moved the lines below into the comments for each method, but these introductory lines, I think, are lost. You can put them right after the group opening, like this:

    /// @{
    /// @name Utility Methods
    ///
    /// Utility methods for ... this paragraph will appear in the right place to
    /// introduce the "group."

Reply:
done.